english/quit-your-job/quit-your-job.md

```diff
@@ -1,2 +1,3 @@
 You suddenly decide that coding and enjoying your life are far mor important than saving money. 
-You decide that you are going to take the leap and quit your job in December 2015. 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.